tests/unit/imports/lib/relativeTimeFormat.ts

```diff
@@ -1,46 +1,56 @@
 import { assert } from 'chai';
-import relativeTimeFormat from '../../../../imports/lib/relativeTimeFormat';
+import { complete, RelativeTimeFormatOpts } from '../../../../imports/lib/relativeTimeFormat';
+
+const verifyFormat = (date: Date, opts: RelativeTimeFormatOpts, expected: string) => {
+  const { formatted, millisUntilChange } = complete(date, opts);
+  assert.equal(formatted, expected);
+  assert.notEqual(millisUntilChange, 0, 'millisUntilChange should never be zero');
+  const justBefore = new Date(date.getTime() - millisUntilChange + 1);
```

Comment:
This feels weirdly structured to me -- shouldn't this be comparing the same `date` against `now + millisUntilChange - 1`, instead of shuffling the reference date around?  (I understand they're ~computationally equivalent, but reading through it was pretty confusing to me, since in usage the reference date is generally the one staying fixed and `now` is generally the thing changing.)

Ditto `justAfter` below -- it'd feel more obviously like we're testing the specific case we expect to encounter if it were `now + millisUntilChange` instead.

Reply:
Fair. I was trying to avoid having to dig into `opts` but you're right. I'll switch